__tests__/app.test.js

```diff
@@ -23,12 +34,21 @@ describe("/api/topics", () => {
         });
       });
   });
-  test("Status: 404 with an error message for if the endpoint the user is searching for doesn't exist in the database", () => {
-    return request(app)
-      .get("/api/invalid-endpoint")
-      .expect(404)
-      .then(({ body }) => {
-        expect(body.msg).toBe("Sorry, request invalid..");
-      });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("Status: 200 GETS an article object with the correct properties", async () => {
+    const { body } = await request(app).get("/api/articles/1").expect(200);
+    expect(typeof body).toBe("object");
+    expect(body.author).toEqual(expect.any(String));
+    expect(body.title).toEqual(expect.any(String));
+    expect(body.article_id).toEqual(expect.any(Number));
+    expect(body.topic).toEqual(expect.any(String));
+    expect(body.created_at).toBe("2020-07-09T20:11:00.000Z");
+    expect(body.votes).toEqual(expect.any(Number));
+  });
+  test("Status: 400 recieves an error when a bad request is made", async () => {
```

Comment:
What is the request that is made is of the right type, but refers to an article not present in the database (e.g id 9999 when we know there's only a few in the test db)?